.github/component_owners.yml

```diff
@@ -3,6 +3,7 @@
 # Each component identified by its path prefix has a list of users
 # Please add the entries in alphabetically sorted order
 components:
+  src/OpenTelemetry.Appenders.Serilog/:
```

Comment:
Remove for now, to make CI working.
When added, it should contain also tests section.

```suggestion
```